src/components/policy/policy_regular/test/policy_manager_impl_test.cc

```diff
@@ -209,7 +209,7 @@ class PolicyManagerImplTest2 : public ::testing::Test {
     ON_CALL(policy_settings_, app_storage_folder())
         .WillByDefault(ReturnRef(kAppStorageFolder));
     manager->set_listener(&listener);
-    const char* levels[] = {"BACKGROUND", "FULL", "LIMITED", "NONE"};
+    const char* levels[] = {"FULL", "LIMITED", "BACKGROUND", "NONE"};
```

Comment:
Was there a reason the ordering of our enum was changed?

Reply:
@mrapitis returned changes back. 66e229195